src/components/ClipboardButton/__tests__/ClipboardButton.visual.test.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import {test} from '~playwright/core';
+
+import {ClipboardButtonStories} from './helpersPlaywright';
+
+const mountOptions = {};
+
+test.describe('ClipboardButton', () => {
+    test('render story: <Default>', async ({mount, expectScreenshot}) => {
+        await mount(<ClipboardButtonStories.Default />, mountOptions);
+
+        await expectScreenshot();
+    });
+
+    test('render story: <CustomTooltipText>', async ({mount, expectScreenshot}) => {
```

Comment:
While tooltip not shown screenshots are the same as default. Let's either remove this story from test or active the tooltip and capture it with the button